packages/core/src/index.js

```diff
@@ -1,6 +1,34 @@
 /** @license MIT License (c) copyright 2016 original author or authors */
-/* eslint-disable import/first */
```

Comment:
I'm in favor of not deleting this one, and keeping imports near their respective exports _in this particular file_.  I think it helps a good bit with readability.  Thoughts?

Reply:
Personally I am in favor of placing curry at the module boundary. 
That would allow the `index.js` to be readable and conform to standard with no overrides.
If we know that there is a significant performance benefit I guess that there is no option for that. 

In any case I don't like code standards overrides and bypasses (although I am not in favor of standard at all but since that decision is made I would like to avoid overrides and follow it all the time)

But if you feel strong about it I can revert this

Reply:
https://jsperf.com/curry-vs-nocurry/1